docs/sources/reference/components/otelcol/otelcol.processor.interval.md

```diff
@@ -0,0 +1,169 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.interval/
+aliases:
+  - ../otelcol.processor.interval/ # /docs/alloy/latest/reference/otelcol.processor.interval/
+description: Learn about otelcol.processor.interval
```

Comment:
```suggestion
description: Learn about otelcol.processor.interval
```

We don't need aliases set on new topics. This should only be used if we move or rename a topic.